src/AgentHeaderDescriptor.php

```diff
@@ -43,44 +41,74 @@ class AgentHeaderDescriptor
     // TODO(michaelbausor): include bumping this version in a streamlined
     // release process. Issue: https://github.com/googleapis/gax-php/issues/48
     const GAX_VERSION = '0.7.0';
+    const UNKNOWN_HEADER = '';
```

Comment:
Let's rename this to `UNKNOWN_VERSION`. Sorry I didn't catch this in my first review.